src/renderer/components/+workloads-cronjobs/cronjobs.tsx

```diff
@@ -8,20 +8,15 @@ import "./cronjobs.scss";
 import React from "react";
 import { observer } from "mobx-react";
 import type { RouteComponentProps } from "react-router";
-import { CronJob, cronJobApi } from "../../../common/k8s-api/endpoints/cron-job.api";
-import { MenuItem } from "../menu";
-import { Icon } from "../icon";
+import type { CronJob } from "../../../common/k8s-api/endpoints/cron-job.api";
 import { cronJobStore } from "./cronjob.store";
 import { jobStore } from "../+workloads-jobs/job.store";
 import { eventStore } from "../+events/event.store";
-import type { KubeObjectMenuProps } from "../kube-object-menu";
 import { KubeObjectListLayout } from "../kube-object-list-layout";
-import { CronJobTriggerDialog } from "./cronjob-trigger-dialog";
 import { KubeObjectStatusIcon } from "../kube-object-status-icon";
-import { ConfirmDialog } from "../confirm-dialog/confirm-dialog";
-import { Notifications } from "../notifications/notifications";
 import type { CronJobsRouteParams } from "../../../common/routes";
 import moment from "moment";
+import { CronJobMenu } from "./cron-job-menu";
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency